src/debugSession.ts

```diff
@@ -66,6 +66,19 @@ export class JanusDebugSession extends DebugSession {
         this.variablesMap = new VariablesMap();
     }
 
+    protected async logVersionNumber() {
```

Comment:
Can we rename this method to `logServerVersion`? Would be more descriptive of what the method actually does.